tasks/system_probe.py

```diff
@@ -1595,11 +1595,25 @@ def print_failed_tests(_, output_dir):
 
 @task
 def save_test_dockers(ctx, output_dir, arch, windows=is_windows):
-    import yaml
-
     if windows:
         return
 
+    images = _test_docker_image_list()
+    for image in images:
+        output_path = image.translate(str.maketrans('', '', string.punctuation))
+        ctx.run(f"docker pull --platform linux/{arch} {image}")
+        ctx.run(f"docker save {image} > {os.path.join(output_dir, output_path)}.tar")
```

Comment:
do we still need to save the image?

Reply:
Yes, because KMT is still using the previous method for the moment. We are working on optimizing KMT as well.